crawler/tasks.py

```diff
@@ -40,38 +44,49 @@ def delete_progress(domain, id):
 @app.task
 def crawl_comic(url):
     item = item_from_url(url)
-    storage = Storage(item.domain, item.id)
-    spider = SPIDERS[item.domain](url)
+    _domain = item.domain
+    _id = item.id
```

Comment:
its ok but just curious.
what's the difference between using the dot operator and assigning it to a tmp variable.

Reply:
It's because `item = spider.crawl(item=item)` may return None on failure, but we need `domain` and `id` to `delete_progress` after that.

Reply:
i see.